src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -346,12 +352,18 @@ private static object GetTypedProperty(IPublishedContent content, Type type)
                                         var converter = DependencyResolver.Current.GetService(toConvert) as TypeConverter;
                                         if (converter != null)
                                         {
+                                            // Create context to pass to converter implementations.
+                                            // This contains the IPublishedContent and the currently converting property name.
+                                            var context = new PublishedContentContext(content, actualPropertyName);
+
+                                            // TODO: Should we pass the current culture when converting?
+                                            object converted = converter.ConvertFrom(context, null, propertyValue);
+
```

Comment:
Yeah, I think it would be useful to pass in the current culture, (it's better than having a `null` right?)